xbmc/music/MusicDatabase.cpp

```diff
@@ -4859,54 +4916,80 @@ bool CMusicDatabase::GetSongsFullByWhere(const std::string &baseDir, const Filte
   {
     unsigned int time = XbmcThreads::SystemClockMillis();
     int total = -1;
+    bool extended = false;
 
     Filter extFilter = filter;
     CMusicDbUrl musicUrl;
     SortDescription sorting = sortDescription;
     if (!musicUrl.FromString(baseDir) || !GetFilter(musicUrl, extFilter, sorting))
       return false;
 
-    // if there are extra WHERE conditions we might need access
-    // to songview for these conditions
+    // If there are extra WHERE conditions (from media filter dialog) we might 
```

Comment:
remove the white space at the end of the line

Reply:
whitespace fixed